publishing/src/main/kotlin/org/projectnessie/buildtools/publishing/PublishingHelperPlugin.kt

```diff
@@ -92,7 +94,11 @@ class PublishingHelperPlugin : Plugin<Project> {
 
                         val parentNode = projectNode.appendNode("parent")
                         parentNode.appendNode("groupId", parent!!.group)
-                        parentNode.appendNode("artifactId", parent!!.mavenArtifactId())
+                        if (e.useMavenArtifactIdProperty.get()) {
```

Comment:
wondering: why is it better to have a separate property for controlling this instead of using `mavenArtifactId` if present but the project name as a fallback automatically?

Reply:
See `mavenArtifactId` - it's computed from parent projects.